droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -271,8 +276,26 @@ def get_rgb_depth(self):
         Returns:
             an RGBDepth object
         """
+        base_state = self.bot.get_base_state()
         rgb, depth, rot, trans = self.cam.get_pcd_data(rotate=False)
+        rgb, depth = jpg_decode(rgb), blosc_decode(depth)
+        depth = np.divide(depth, 1000, dtype=np.float32)  # convert from mm to metres
+        base_state = self.bot.get_base_state().value
+        uv_one_in_cam = self.uv_one_in_cam
+        return HelloRobotMover.compute_pcd(rgb, depth, rot, trans, base_state, uv_one_in_cam)
+
+    @staticmethod
+    def compute_uvone(intrinsic_mat, height, width):
```

Comment:
probably better to keep these as util methods longer term instead of static methods, so any robot can call into them (for eg both locobot and hello do the same thing now). Can happen after this PR.